utils/src/yaml_parser.cpp

```diff
@@ -941,7 +909,60 @@ int32_t get_classname(char *model_path, char (*classnames)[256])
             name = data["supercategory"].as<std::string>() + "/" + name;
         }
 
-        sprintf(classnames[id], name.data());
+        if(id < (int32_t) max_classname)
+        {
+            sprintf(classnames[id], name.data());
```

Comment:
Use snprintf to only copy 64 byte max, since size of classnames[i] is 64

Reply:
Done